spanner/spanner_snippets/spanner/spanner_create_instance_config.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_create_instance_config]
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	instancepb "google.golang.org/genproto/googleapis/spanner/admin/instance/v1"
+)
+
+// createInstanceConfig creates a custom spanner instance config
+func createInstanceConfig(w io.Writer, projectPath string, userConfigID, baseConfigPath string) error {
```

Comment:
Just wondering if here (and elsewhere) you want to follow the common practice of accepting IDs instead of paths, and add the static elements of the paths in the sample. This is done for example with `projectID` and `instanceID` in [spanner_create_instance.go](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/spanner_create_instance.go#L28-L41).

Reply:
Thanks for pointing this out, done with all changes.